From c0167f0c0c0ede02dd3d8a6d61c73c2b1631d465 Mon Sep 17 00:00:00 2001
From: Jason Hrycay <jason.hrycay@motorola.com>
Date: Thu, 15 May 2014 15:59:53 -0500
Subject: [PATCH 751/959] IKSWM-24205 fs: pstore: provide inline pstore_annoate

Add an inline dummy function when PSTORE is disabled so that we can
compile even when there are still other users of it.

Change-Id: Ie899db5572d1e4837e46027b00af0d16023404c6
Signed-off-by: Jason Hrycay <jason.hrycay@motorola.com>
Reviewed-on: http://gerrit.mot.com/830273
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
---
 include/linux/pstore.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/linux/pstore.h b/include/linux/pstore.h
index 29a89b28a50..88569ab0266 100644
--- a/include/linux/pstore.h
+++ b/include/linux/pstore.h
@@ -88,6 +88,11 @@ pstore_cannot_block_path(enum kmsg_dump_reason reason)
 {
 	return false;
 }
+static inline int
+pstore_annotate(const char *buf)
+{
+	return -ENODEV;
+}
 #endif
 
 #endif /*_LINUX_PSTORE_H*/
-- 
2.11.0

